app/src/test/java/org/simple/clinic/newentry/form/DateOfBirthFieldTest.kt

```diff
@@ -0,0 +1,22 @@
+package org.simple.clinic.newentry.form
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+import org.simple.clinic.newentry.form.ValidationError.InvalidDateFormat
+import org.simple.clinic.newentry.form.ValidationError.MissingValue
+
+class DateOfBirthFieldTest {
```

Comment:
Should we also add a test that entering a date in a valid format does not throw any validation error?

Reply:
Yes, that is happening in one of the upcoming commits.